INFO - 2023-06-13 11:24:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:24:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:25:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-06-13 11:25:20 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-13 11:25:20 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab10\\public\\index.php')
ERROR - 2023-06-13 11:28:48 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#5 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#6 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#7 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#8 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#4 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#5 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#6 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#7 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#8 {main}
CRITICAL - 2023-06-13 11:28:48 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 SYSTEMPATH\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(null)
 4 SYSTEMPATH\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
 5 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run([...])
 6 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', [...])
 7 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
INFO - 2023-06-13 11:30:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-06-13 11:30:29 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-13 11:30:29 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab10\\public\\index.php')
INFO - 2023-06-13 11:30:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:30:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-06-13 11:30:38 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-13 11:30:38 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab10\\public\\index.php')
INFO - 2023-06-13 11:31:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:31:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:31:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-06-13 11:31:37 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-13 11:31:37 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab10\\public\\index.php')
ERROR - 2023-06-13 11:37:47 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#5 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#6 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#7 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#8 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#4 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#5 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#6 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#7 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#8 {main}
CRITICAL - 2023-06-13 11:37:47 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 SYSTEMPATH\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(null)
 4 SYSTEMPATH\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
 5 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run([...])
 6 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', [...])
 7 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
INFO - 2023-06-13 11:38:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:38:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:38:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-06-13 11:38:53 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-13 11:38:53 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab10\\public\\index.php')
INFO - 2023-06-13 11:44:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:44:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 11:44:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-06-13 11:44:54 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-13 11:44:54 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab10\\public\\index.php')
ERROR - 2023-06-13 11:53:42 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#5 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#6 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#7 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#8 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#4 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#5 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#6 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#7 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#8 {main}
CRITICAL - 2023-06-13 11:53:42 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 SYSTEMPATH\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(null)
 4 SYSTEMPATH\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
 5 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run([...])
 6 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', [...])
 7 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
ERROR - 2023-06-13 11:56:50 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#5 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#6 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#7 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#8 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#4 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#5 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#6 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#7 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#8 {main}
CRITICAL - 2023-06-13 11:56:50 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 SYSTEMPATH\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(null)
 4 SYSTEMPATH\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
 5 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run([...])
 6 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', [...])
 7 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
ERROR - 2023-06-13 12:00:51 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#5 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#6 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#7 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#8 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#4 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#5 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#6 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#7 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#8 {main}
CRITICAL - 2023-06-13 12:00:51 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 SYSTEMPATH\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(null)
 4 SYSTEMPATH\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
 5 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run([...])
 6 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', [...])
 7 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
ERROR - 2023-06-13 12:03:22 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#5 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#6 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#7 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#8 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\lab10\system\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(NULL)
#4 C:\xampp\htdocs\lab10\system\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#5 C:\xampp\htdocs\lab10\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run(Array)
#6 C:\xampp\htdocs\lab10\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', Array)
#7 C:\xampp\htdocs\lab10\spark(97): CodeIgniter\CLI\Console->run()
#8 {main}
CRITICAL - 2023-06-13 12:03:22 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 SYSTEMPATH\Database\Seeder.php(97): CodeIgniter\Database\Config::forge(null)
 4 SYSTEMPATH\Commands\Database\Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
 5 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Seed->run([...])
 6 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('db:seed', [...])
 7 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
INFO - 2023-06-13 12:03:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 12:03:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 12:03:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-06-13 12:04:00 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-13 12:04:00 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab10\\public\\index.php')
INFO - 2023-06-13 12:04:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 12:04:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-06-13 12:04:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-06-13 12:04:28 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\lab10\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab10\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab10\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab10\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab10\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab10\app\Controllers\User.php(26): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab10\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab10\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab10\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab10\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab10\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-13 12:04:28 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(26): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab10\\public\\index.php')
